### PR TITLE
changed view of production module #7083

### PR DESCRIPTION
**Pull request by vishu1993**

### CODE REVIEW

**Comment by vishu1993** [comment]
productionneatview

**Comment by PritishC** [comment]
Can you separate this into two commits?

One will have regular boilerplate.
One will have the changes you made to your view.

**Comment by PritishC** [comment]
LGTM.

File: view/production_form.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+<data>
+    <xpath expr="/form/label[@name='reference']" position="replace">
+    </xpath>
+    <xpath expr="/form/field[@name='reference']" position="replace">
+    </xpath>
+    <xpath expr="/form/field[@name='code']" position="replace">
+        <group col="8">
+            <label name="code" />
+            <field name="code" colspan="1"/>
+	    <label name="reference" />
```
Comment (tarunbhardwaj) [inline view/production_form.xml]: fix intentation

File: view/production_form.xml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            <field name="reference" colspan="1"/>
+   	    <label name="planned_date" />
```
Comment (tarunbhardwaj) [inline view/production_form.xml]: fix intentation

**Comment by vishu1993** [comment]
production_neat

**Comment by PritishC** [comment]
The date fields look a little awkward (too short). It is probably a colspan issue. Could you check please?

**Comment by vishu1993** [comment]
productionfinal

**Comment by vishu1993** [comment]
productionfinal

File: view/production_form.xml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    </xpath>
+    <xpath expr="/form/label[@name='planned_date']" position="replace">
+    </xpath>
+    <xpath expr="/form/field[@name='planned_date']" position="replace">
+    </xpath>
+    <newline/>
+    <xpath expr="/form/label[@name='effective_date']" position="replace">
+    </xpath>
+    <xpath expr="/form/field[@name='effective_date']" position="replace">
+        <group colspan="8" col="4">
```
Comment (PritishC) [inline view/production_form.xml]: `id` or `name` required.

File: view/production_form.xml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+            <field name="warehouse"/>
+            <label name="location"/>
+            <field name="location"/>
+        </group>
+    </xpath>
+    <newline/>
+    <xpath expr="/form/label[@name='product']" position="replace">
+    </xpath>
+    <xpath expr="/form/field[@name='product']" position="replace">
+        <group col="8">
```
Comment (PritishC) [inline view/production_form.xml]: `id` or `name` required.

**Comment by PritishC** [comment]
LGTM

**Comment by vishu1993** [comment]
production_neat_view

**Pull request merged**